docs/source/release/v3.10.0/framework.rst

```diff
@@ -17,6 +17,8 @@ Algorithms
 ----------
 
 - Removed the optional flag ``LocationParameters`` from ``ClearInstrumentParameters``.
+- Override method `helpURL` to return the webpage for the algorithm's documentation. Useful when registering Python
```

Comment:
Well that is clear, but from the sentence it is not obvious that it is a method of the base algorithm class, that could be overridden.